vulcanai/model_tests.py

```diff
@@ -187,18 +189,18 @@ def run_test(network, test_x, test_y, figure_path='figures', plot=True):
     print ('Average AUC: : {:.4f}'.format(np.average(all_class_auc)))
     return {
         'accuracy': accuracy,
-        'macro_sensitivity': sens_macro,
-        'macro_specificity': spec_macro,
+        'sensitivity': sens[0],
```

Comment:
Because the macro metrics are a weighted average of the all the classes which is always equal to the accuracy and doesn't distinguish between the classes

Reply:
The code I've written there is just temporary for a binary class case, it will need to be modified to fit the general n class case